vcr/api/vcr/v2/types.go

```diff
@@ -21,37 +21,37 @@ package v2
 
 import (
 	"encoding/json"
-	"github.com/nuts-foundation/go-did/vc"
 	"github.com/nuts-foundation/nuts-node/vcr/credential"
+	"github.com/nuts-foundation/nuts-node/vcr/types"
 )
 
-// VerifiableCredential is an alias to use from within the API
-type VerifiableCredential = vc.VerifiableCredential
+var _ json.Unmarshaler = &types.CompactingVerifiableCredential{}
+var _ json.Marshaler = &types.CompactingVerifiableCredential{}
 
 // CredentialSubject is an alias to use from within the API
 type CredentialSubject = interface{}
 
 // Revocation is an alias to use from within the API
 type Revocation = credential.Revocation
 
-// VerifiablePresentation is an alias to use from within the API
-type VerifiablePresentation = vc.VerifiablePresentation
+var _ json.Unmarshaler = &types.CompactingVerifiablePresentation{}
+var _ json.Marshaler = types.CompactingVerifiablePresentation{}
```

Comment:
these should be in the file declaring `CompactingVerifiableCredential` and `CompactingVerifiablePresentation`

Reply:
Left them there for now until we resolve the go-leia discussion